### PR TITLE
Sus 957 - changed toolbar label from Follow to Watch

### PR DESCRIPTION
**Pull request by ukaka**
https://wikia-inc.atlassian.net/browse/SUS-957

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/toolbars/CustomizedToolbarComponentObject.java [hunk 1]
```diff
@@ -151,30 +151,6 @@ public void verifyFollowMessage() {
   }
 
   /**
-   * Verify that page is followed The method should be used only after clicking on "follow" button.
-   * Before that, "follow" button does not have 'title' attribute which is necessary in the method
-   */
```
Comment (ukaka) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/toolbars/CustomizedToolbarComponentObject.java]: I removed the method as Watch and Unwatch buttons on toolbar don't have `title` attribute anymore (for both states, before and after click). Method was used only in `CustomizeToolbar006_ButtonsActions` test.
There is another method in test which checks if button label was correctly changed after clicking.

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/toolbars/CustomizedToolbarComponentObject.java [hunk 2]
```diff
@@ -151,30 +151,6 @@ public void verifyFollowMessage() {
...
-    PageObjectLogging.log("verifyFollowedToolbar", "follow button verified", true);
-
-  }
-
-  /**
-   * Verify that page is unfollowed The method should be used only after clicking on "Unfollow"
-   * button. Before that, "follow" button does not have 'title' attribute which is necessary in the
-   * method
-   */
-  public void verifyUnfollowed() {
```
Comment (ukaka) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/toolbars/CustomizedToolbarComponentObject.java]: the same here

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/toolbars/CustomizedToolbarComponentObject.java [hunk 3]
```diff
@@ -151,30 +151,6 @@ public void verifyFollowMessage() {
   }
 
   /**
-   * Verify that page is followed The method should be used only after clicking on "follow" button.
-   * Before that, "follow" button does not have 'title' attribute which is necessary in the method
-   */
-  public void verifyFollowedToolbar() {
```
Comment (ukaka) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/toolbars/CustomizedToolbarComponentObject.java]: I removed the method as Watch and Unwatch buttons on toolbar don't have `title` attribute anymore (for both states, before and after click). Method was used only in `CustomizeToolbar006_ButtonsActions` test.
There is another method in test which checks if button label was correctly changed after clicking.

**Comment by ludwikkazmierczak** [comment]
LGTM

**Pull request merged**